### PR TITLE
Front-end

### PR DESCRIPTION
**Pull request by SevenDart**

### CODE REVIEW

File: CUEstion.BLL/QuestionManagerService.cs [hunk 1]
```diff
@@ -397,5 +397,56 @@ public void MarkComment(int commentId, int mark)
...
+
+
+		public void UpdateTag(string oldTag, string newTag)
+		{
+			using var context = new ApplicationContext();
+
+			var foundTag = context.Tags.FirstOrDefault(t => t.Name.ToLower() == oldTag.ToLower());
+			if (foundTag == null)
+			{
+				throw new Exception();
```
Comment (Mopsist) [inline CUEstion.BLL/QuestionManagerService.cs]: if you want to use exceptions here - it is better to add some error message

File: CUEstion.WEB/ClientApp/src/app/services/questions.service.ts [hunk 2]
```diff
@@ -160,4 +180,18 @@ export class QuestionsService {
...
+
+  DeleteComment(questionId: number, answerId: number, commentId: number) {
+    const options = {
+      headers: new HttpHeaders({
+        Authorization: 'Bearer ' + localStorage.getItem('token')
+      })
+    };
+
+    if (answerId != null) {
+      return this.http.delete(this._serverAddress + `/questions/0/answers/${answerId}/comments/${commentId}`, options);
```
Comment (Mopsist) [inline CUEstion.WEB/ClientApp/src/app/services/questions.service.ts]: probably I am missing something. can you explain me why "0" is in the route here?

File: CUEstion.WEB/ClientApp/src/app/services/questions.service.ts [hunk 3]
```diff
@@ -75,6 +78,86 @@ export class QuestionsService {
...
+    };
+
+    const options = {
+      headers: new HttpHeaders({
+        Authorization: 'Bearer ' + localStorage.getItem('token')
+      })
+    };
+
+    if (answerId != null) {
+      return this.http.post(this._serverAddress + `/questions/0/answers/${answerId}/comments`, comment, options);
```
Comment (Mopsist) [inline CUEstion.WEB/ClientApp/src/app/services/questions.service.ts]: the same question. why is 0 here in the route?

File: CUEstion.WEB/ClientApp/src/app/services/questions.service.ts (hunk 2 above)
Comment (SevenDart) [reply CUEstion.WEB/ClientApp/src/app/services/questions.service.ts]: I've written 0 cause it doesn't matter, what number of question is, cause questions, answers, comments have their different unique Ids. In places, where I call such methods I do this due to lack of opportunity to give this methods necessary parameters, although they are optional

**Pull request closed without merging**